src/index.js

```diff
@@ -97,7 +97,7 @@ class TextEllipsis extends React.Component {
             this.container = node;
           },
           className: this.props.tagClass,
-          style: { width: '100%', wordWrap: 'break-word' },
+          style: Object.assign({ width: '100%', wordWrap: 'break-word' }, this.props.style})
```

Comment:
please remove `}` at the end, it is getting compile error
https://travis-ci.org/tjindapitak/react-text-ellipsis/builds/328560240?utm_source=github_status&utm_medium=notification